### DISCUSSION TITLE
Display on the GUI the content of the log file.

### DISCUSSION THREAD

**Discussion by sebo83910** [TomSchimansky/CustomTkinter#461] [Q&A]
Hello,

First of all, thanks for the good work. I've been playing around with some of your examples.
My project analyses some data and creates some print out.
I'd like to display on the GUI the content of the log file.
Is that possible?

Thanks.
Seb.

**Comment by TomSchimansky** [comment]
Hey, therefore you would use a textbox, the CTkTextbox is currently under development but should be ready in the next time. For now you could use a normal tkinter textbox if you want.
